docs/products/m3db/howto/monitoring.rst

```diff
@@ -0,0 +1,14 @@
+Monitor Aiven Services with M3DB
+================================
+
+M3DB is a perfect fit for a monitoring platform. It is designed to handle large volumes of metrics, and it's very good at that. Whether you are already working at scale or just curious about M3DB and monitoring, setting up an M3DB to monitor your existing Aiven services (or new ones if you like) is a really nice way to get started with this platform.
+
+Start from the service overview page of the service you would like to monitor with M3. So if you're going to track the metrics of your PostgreSQL service on Aiven, then start on the PostgreSQL service overview page.
+
+1. Under "Manage Integrations", look for "Metrics" (not available on all services yet).
```

Comment:
Adding the image of the Manage Integration screen will help readers understand if they are in the good place and what to look for?

Reply:
I dislike screenshots in general because they are hard to maintain - and the integrations screen definitely should change from time to time so I'm going to leave it out.